src/expander_pipeline.jl

```diff
@@ -774,13 +774,25 @@ end
 # @example
 # --------
 
-# Find if there is any format with color output
+# Find if there is any format with color output.
 function _any_color_fmt(doc)
-    idx = findfirst(x -> x isa Documenter.HTML, doc.user.format)
+    idx = findfirst(writer_supports_ansicolor, doc.user.format)
     idx === nothing && return false
     return doc.user.format[idx].ansicolor
 end
 
+# General fallback.
+"""
+    writer_supports_ansicolor(::Documenter.Writer)::Bool
+	
```

Comment:
There's some trailing whitespace in this line (the GitHub UI doesn't show it)

@mortenpi Don't we have automatic code formatting in place at this point? Source code formatting, including preventing trailing white space should be be flagged by CI

Reply:
There is a Runic run that seems to have passed, maybe Runic doesn't pick this up?

https://github.com/JuliaDocs/Documenter.jl/actions/runs/14786412938/job/41515527745?pr=2490

Reply:
Well, now I think you deleted the blank line that makes the signature a code block.

There is also more trailing whitespace in the last line of that docstring.

Independent of `runic` picking this up: I would recommend setting up your text editor to remove trailing spaces from `.jl` files automatically, on save.

We should probably also have a `make codestyle` or something that people can use to easily ensure that all code is properly formatted when preparing PRs.

Reply:
I think I was able to push a fix-commit to this PR…